{{ cookiecutter.format }}/briefcase.toml

```diff
@@ -3,13 +3,12 @@
 app_path = "{{ cookiecutter.formal_name }}.AppDir/usr/app"
 app_packages_path = "{{ cookiecutter.formal_name }}.AppDir/usr/app_packages"
 support_path = "{{ cookiecutter.formal_name }}.AppDir/usr"
-{# using 20230826 until indygreg/python-build-standalone#194 is resolved -#}
 {{ {
-    "3.8": 'support_revision = "3.8.17+20230826"',
-    "3.9": 'support_revision = "3.9.18+20230826"',
-    "3.10": 'support_revision = "3.10.13+20230826"',
-    "3.11": 'support_revision = "3.11.5+20230826"',
-    "3.12": 'support_revision = "3.12.0+20231002"',
+    "3.8": 'support_revision = "3.8.18+20240107"',
+    "3.9": 'support_revision = "3.9.18+20240107"',
+    "3.10": 'support_revision = "3.10.13+20240107"',
+    "3.11": 'support_revision = "3.11.7+20240107"',
+    "3.12": 'support_revision = "3.12.1+20231002"',
```

Comment:
Is the lack of datestamp bump intentional? 
```suggestion
    "3.12": 'support_revision = "3.12.1+20240107"',
```

Reply:
👀 No; good catch.

However, why didn't CI fail for this error??? It certainly should have...

hmm....CI is _always_ using Python 3.10....all right; that's a bigger problem....

https://github.com/beeware/briefcase-linux-appimage-template/actions/runs/7450534729/job/20269664684#step:21:128

Reply:
The logic in `app-build-verify` for whether to run `actions/setup-python` or not will need to be adjusted since anytime the workflow runs currently, it _always_ uses Python 3.10 if the runner is Ubuntu. So, this also affects Flatpak tests.

It might be possible to simply unset the environment variables for Linux System builds...or something...

Reply:
Addressed via https://github.com/beeware/.github/pull/79